### PR TITLE
Add some sql types

### PR DESCRIPTION
**Pull request by tgallice**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module github.com/upfluence/sql
+
+go 1.13
+
+replace github.com/coreos/bbolt v1.3.4 => go.etcd.io/bbolt v1.3.4
```
Comment (AlexisMontagne) [inline go.mod]: why?

File: sqltype/null_utc_time.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package sqltype
```
Comment (AlexisMontagne) [inline sqltype/null_utc_time.go]: i thought we said `sqltypes` ?

Comment (tgallice) [reply sqltype/null_utc_time.go]: yes but to be consistent we others folders which are in singular, no ?

File: go.mod (hunk 1 above)
Comment (tgallice) [reply go.mod]: needed for github.com/upfluence/cfg

File: sqltype/null_utc_time.go (hunk 2 above)
Comment (AlexisMontagne) [reply sqltype/null_utc_time.go]: it is like, `errors` or `strings` when it it is a suite of tool without a main purpose ( a specific struct ) you can pluralize

**Review by AlexisMontagne — APPROVED** [review]

**Pull request merged**